lms/djangoapps/program_enrollments/api/tests/test_linking.py

```diff
@@ -256,6 +256,45 @@ def test_realize_course_access_roles_user_with_existing_course_access_role(self)
         # assert that all CourseAccessRoleAssignment objects are deleted
         assert not active_enrollment_1.courseaccessroleassignment_set.all().exists()
 
+    def test_enrollment_already_linked_to_different_user(self):
```

Comment:
The function name can be clearer with `test_update_linking_enrollment_to_another_user`